src/api/index.ts

```diff
@@ -116,26 +116,53 @@ function createTokenListApi(): TokenList {
 }
 
 function createTheGraphApi(): TheGraphApi {
-  const theGraphApi = new TheGraphApiProxy({ config: config.theGraphApi })
+  const { type, config: theGraphApiConfig } = CONFIG.theGraphApi
```

Comment:
nit: It's scoped inside the function, why not just use `config`?

Reply:
good point, I think I had a global config before, and just wanted to go specific here. Having `config` and `CONFIG` might be slightly confusing, but I'm with you that it pays off by being simpler